lib/semantics/check-do.cc

```diff
@@ -213,9 +235,9 @@ class DoConcurrentBodyEnforce {
     return false;
   }
 
-  void addName(const std::optional<parser::Name> &nm) {
-    if (nm.has_value()) {
-      names_.insert(nm.value().source);
+  void addName(NamePtr nm) {
```

Comment:
`addName` is not an accessor, mutator, or obvious analog to a feature in a standard library, so it should be capitalized.

Reply:
Thanks.  I'll make that change.